repoclosure/yum.conf

```diff
@@ -173,6 +154,6 @@ failovermethod=priority
 name=f29-puppet-6
 baseurl=http://yum.puppetlabs.com/puppet6/fedora/29/x86_64/
 
-[f29-foreman-client-nightly]
-name=Foreman Client nightly F29
-baseurl=http://koji.katello.org/releases/yum/foreman-client-nightly/fc29/x86_64/
+#[f29-foreman-client-nightly]
+#name=Foreman Client nightly F29
+#baseurl=http://koji.katello.org/releases/yum/foreman-client-nightly/fc29/x86_64/
```

Comment:
We can safely drop this and all other f29 (and in nightly too) bits.

Reply:
Perfect, thanks!